### PR TITLE
WPF-876347-Updated the UG documentation for the newly implemented features

### PR DESCRIPTION
**Pull request by Karthick-Arjunan**
### Description
Added the UG documentation for the newly implemented show/Hide the RadialgradientBrush properties in the color picker and NewButtonClick event in the docking manager.

### CODE REVIEW

File: wpf/Docking/MDI-TDI-functionalities.md [hunk 1]
```diff
@@ -746,6 +746,63 @@ In TDI document, new tab group can be created by dragging the tab item into the
...
+{% highlight C# %}
+
+//Adding new button in docking manager using C#
+
+EventManager.RegisterClassHandler(typeof(DocumentTabControl), LoadedEvent, new RoutedEventHandler(DocumentTabControl_Loaded), true);
+
+private void DocumentTabControl_Loaded(object sender, RoutedEventArgs e)
+{
+    DocumentTabControl tab = sender as DocumentTabControl;
+    tab.IsNewButtonEnabled = true;
```
Comment (durga-muthu) [inline wpf/Docking/MDI-TDI-functionalities.md:784]: Already style has been provided. then why we are setting the properties in loaded?

File: wpf/Docking/MDI-TDI-functionalities.md [hunk 2]
```diff
@@ -746,6 +746,63 @@ In TDI document, new tab group can be created by dragging the tab item into the
...
+    </syncfusion:DockingManager.DocumentTabControlStyle>
+</syncfusion:DockingManager>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+//Adding new button in docking manager using C#
+
+EventManager.RegisterClassHandler(typeof(DocumentTabControl), LoadedEvent, new RoutedEventHandler(DocumentTabControl_Loaded), true);
```
Comment (durga-muthu) [inline wpf/Docking/MDI-TDI-functionalities.md:779]: remove this code.

File: wpf/Docking/MDI-TDI-functionalities.md [hunk 3]
```diff
@@ -746,6 +746,63 @@ In TDI document, new tab group can be created by dragging the tab item into the
 
 ![WPF Docking First Item Inserted to Next Tab Group by using Context Menu](MDI_TDIfunctionalities_images/wpf-docking-first-item-inserted-to-next-tab-group-by-using-context-menu.jpeg)
 
+### Adding a new Tab using NewButtonClick event
+
+The NewButtonClick event is used to notifies when the new button(+) is clicked in DockingManager. In TDI document, new tab can be added at runtime by utilizing this event.
```
Comment (durga-muthu) [inline wpf/Docking/MDI-TDI-functionalities.md:751]: TDI is for tabbed document interface, please use correct sentence here.

**Review by durga-muthu — CHANGES_REQUESTED** [review]

File: wpf/Docking/MDI-TDI-functionalities.md (hunk 1 above)
Comment (Karthick-Arjunan) [reply wpf/Docking/MDI-TDI-functionalities.md:784]: I have provided code snippet for adding the new button through the code behind.

File: wpf/Docking/MDI-TDI-functionalities.md (hunk 2 above)
Comment (Karthick-Arjunan) [reply wpf/Docking/MDI-TDI-functionalities.md:779]: removed.

File: wpf/Docking/MDI-TDI-functionalities.md (hunk 3 above)
Comment (Karthick-Arjunan) [reply wpf/Docking/MDI-TDI-functionalities.md:751]: Changed the content.

**Review by durga-muthu — APPROVED** [review]

**Pull request merged**